packages/account/src/BiconomySmartAccount.ts

```diff
@@ -256,6 +256,37 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
     return "0x";
   }
 
+  private async getNonce(): Promise<BigNumber> {
+    let nonce = BigNumber.from(0);
+    try {
+      nonce = await this.nonce();
+    } catch (error) {
+      // Not throwing this error as nonce would be 0 if this.nonce() throw exception, which is expected flow for undeployed account
+    }
+    return nonce;
+  }
+
+  private async getGasFeeValues(
+    overrides: Overrides | undefined,
+    skipBundlerGasEstimation: boolean | undefined,
+  ): Promise<{ maxFeePerGas?: BigNumberish | undefined; maxPriorityFeePerGas?: BigNumberish | undefined }> {
+    const gasFeeValues = {
+      maxFeePerGas: overrides?.maxFeePerGas,
+      maxPriorityFeePerGas: overrides?.maxPriorityFeePerGas,
+    };
+    try {
+      if (this.bundler && !gasFeeValues.maxFeePerGas && !gasFeeValues.maxPriorityFeePerGas && (skipBundlerGasEstimation ?? true)) {
+        const gasFeeEstimation = await this.bundler.getGasFeeValues();
+        gasFeeValues.maxFeePerGas = gasFeeEstimation.maxFeePerGas;
+        gasFeeValues.maxPriorityFeePerGas = gasFeeEstimation.maxPriorityFeePerGas;
+      }
+      return gasFeeValues;
+    } catch (error: any) {
+      Logger.error("Provided bundler might not have this endpoint", error);
```

Comment:
Better logging message should be used.

"Error while getting gasFeeValues from bundler. Provided bundler might not have getGasFeeValues endpoint"